package.json

```diff
@@ -1,159 +1,159 @@
 {
-    "name": "snippet",
-    "displayName": "Snippet",
-    "description": "Insert a snippet from cht.sh for Python, JavaScript, Ruby, C#, Go, Rust (and any other language)",
-    "version": "0.7.0",
-    "publisher": "vscode-snippet",
-    "engines": {
-        "vscode": "^1.34.0"
-    },
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/mre/vscode-snippet.git"
-    },
-    "license": "MIT",
-    "icon": "contrib/icon.png",
-    "galleryBanner": {
-        "color": "#6cfff9",
-        "theme": "light"
-    },
-    "categories": [
-        "Programming Languages",
-        "Snippets",
-        "Other"
-    ],
-    "tags": [
-        "python",
-        "ruby",
-        "php",
-        "rust",
-        "C#",
-        "go",
-        "haskell",
-        "typescript",
-        "shell",
-        "javascript",
-        "node",
-        "snippet",
-        "examples",
-        "documentation",
-        "help",
-        "tldr",
-        "helper",
-        "cheatsheet"
+  "name": "snippet",
+  "displayName": "Snippet",
+  "description": "Insert a snippet from cht.sh for Python, JavaScript, Ruby, C#, Go, Rust (and any other language)",
+  "version": "0.7.0",
+  "publisher": "vscode-snippet",
+  "engines": {
+    "vscode": "^1.39.0"
```

Comment:
That's the only line I changed in this file. Please ignore the rest as it's just formatting.